Resume.php

```diff
@@ -1,34 +1,7 @@
-<html>
-  <head>
-    <title> Resume/Projects </title>
-    <link rel="stylesheet" href="style.css?Time=<?php echo microtime()?>" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  </head>
-  <body>
-      <h1>
-          Projects/Resume</h1>
-          <div class = "active">
-                <ul>
-                <li>
-                    <a href = "index.php">
-                    Home
-                </a> </li>
-                    <li>
-                        <a href = "blogpost1.php">
-                            Contact
-                    </a> </li>
-
-                    <li>
-                        <a href = "Resume.php">
-                         Projects/Resume </a> </li>
-                  <li>
-                      <a href = "blogpost2.php">
-                          Summer Blog </a> </li>
-                </ul>
-          </div>
-
-     <br>
-     <div style='max-width: 90%; background-color: #B3C3E4; padding:10px; margin:auto; border-radius:10px'>
+<?php
+include('config/init.php');
+ headerA("Project/Resume", "Projects and Resumes");
```

Comment:
I assume you're calling this function `headerA` either because (a) there's already a function called `header` or (b) you plan on making more headers and this is just the first variation.

Either way, I'd suggest making the name of this function a bit more descriptive. I know that in computer science classes you're used to using very short variable and function names, but I personally think it's a mistake for them to teach it that way, because it doesn't work well in the real world. When you name a function or a variable, think about what someone might think looking at this years from now without any context. `headerA` might not really mean a lot to them.

If you call this `BlogHeader` that would tell them there are multiple types of header functions, and this one is specifically for the blog.

If you call it `EchoHeaderHtml` that tells them that this function echos html (rather than returning a value or something like that).

If you call it `EchoBlogHeaderHtml` then it says both at the same time.

I'm not necessarily saying you need to go that far, but something more descriptive than `headerA` would probably be a good idea.